data/tribes/initialization/empire/units.lua

```diff
@@ -1138,6 +1142,10 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_headquarters_shipwreck",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Headquarters Shipwreck
+            lore = pgettext("empire_building", "‘Naufragium sed domo.’ (Although it's wrecked, it's still our home)"),
```

Comment:
Use `’` instead of `'` (more below).

Also, shouldn't the English translation end with a period as well? (this also applies to more strings)

Reply:
Sorry, I changed it. I thought the `'` character is proper for this use case, but I guess the issue here is the possible clash with string delimiters.